samples/net/coap_server/src/coap-server.c

```diff
@@ -737,7 +737,7 @@ static int large_get(struct coap_resource *resource,
 	size = min(coap_block_size_to_bytes(ctx.block_size),
 		   ctx.total_size - ctx.current);
 
-	memset(payload, 'A', size);
+	memset(payload, 'A', size > sizeof(payload) ? sizeof(payload) : size);
```

Comment:
We have quite a good use of min() use in the codebase, so please use it here too.